### ISSUE TITLE
[Feature Request] - Please rapidly allow more instance so we can get our 50,000 user instead of 11,250.

### ISSUE THREAD

**Issue by VFXpro** [microsoft/azure-load-testing#116]
We cannot go above 45 instances x 250 users :(  

(actually, as yesterday, we cannot even go about 1 instance)

This product is definitely not ready for prime time with a limit of 11,250 users.  We will need to rapidly find another service somewhere else since this product has been limited since day one and Microsoft refuses to increase this limit. :(

AB#1664230

**Comment by Nagarjuna-Vipparthi** [comment]
@VFXpro Can you please share the info message about the limits that you see on the 'Load' tab of the test creation flow? We can suggest next steps based on that.

250 users per engine is a guidance to keep the engines in healthy state. However, depending on your scenario, you may go for higher count as well. Please refer this blogpost for more details - https://techcommunity.microsoft.com/t5/apps-on-azure-blog/configure-jmeter-script-to-optimize-utilization-of-test-engines/ba-p/3618230

**Comment by VFXpro** [comment]
Hi @Nagarjuna-Vipparthi .  Thanks for your reply.  The problem we have at this specific moment (and Azure support has not responded in 48 hours!!) is that we cannot increase the number of **engine instance higher than 1**.   In the past we were able to increase it to 45... and now we can only use 1.  We URGENTLY needs this problem to be fixed but no one is responding to us.  :(

**Comment by VFXpro** [comment]
This issue is NOT closed...

**Comment by VFXpro** [comment]
Here's a screenshot... pretty simple.  Cannot drag or type to increase the number of instance.
Also, we have created different Azure Load Testing service in different regions, EAST US, CENTRAEL US, WEST US and they all the same limit of 1 instance.  Please do something about as soon as possible...  Thanks.
![image](https://user-images.githubusercontent.com/44643864/199871397-73d9f485-8f25-412b-91d7-3723c94f88a8.png)

**Comment by Nagarjuna-Vipparthi** [comment]
@VFXpro We understand the urgency here. Can you please let us know if you have raised a request to increase the limits? 

If not, please follow the instructions here to raise a request - https://learn.microsoft.com/en-us/azure/load-testing/resource-limits-quotas-capacity#request-quota-increases

**Comment by VFXpro** [comment]
Yes, I completed 3 separate support requests in the last 3 days...no one ever gets back to me.  Really unsatisfied customer.
________________________________

**Comment by Nagarjuna-Vipparthi** [comment]
@VFXpro Can you please provide the ticket numbers so that we can follow up and help you with the resolution?

**Comment by VFXpro** [comment]
Thanks for helping resolve this issue right away.

The case number 2211040040006460
It has BEEN MANY DAYS, and no one is doing anything about.  We can't increase the count of instance bigger than 1.
Also, the current limit of 45 instances is ridiculous, this needs to be addresses ASAP.
________________________________

**Comment by Nagarjuna-Vipparthi** [comment]
> Thanks for helping resolve this issue right away. The case number 2211040040006460 It has BEEN MANY DAYS, and no one is doing anything about. We can't increase the count of instance bigger than 1. Also, the current limit of 45 instances is ridiculous, this needs to be addresses ASAP.
> […](#)
> ________________________________ From: Nagarjuna Vipparthi ***@***.***> Sent: Monday, November 7, 2022 1:41 AM To: microsoft/azure-load-testing ***@***.***> Cc: VFXpro ***@***.***>; Mention ***@***.***> Subject: Re: [microsoft/azure-load-testing] [Feature Request] - Please rapidly allow more instance so we can get our 50,000 user instead of 11,250. (Issue #116) @VFXpro<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FVFXpro&data=05%7C01%7C%7Cf95dffeb90ca4d2a2bdf08dac08b1038%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638034000747948184%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lOLy9bussIVeWHVeQDyg5OHoT3KVWyBOPw0NDWQeK%2F0%3D&reserved=0> Can you please provide the ticket numbers so that we can follow up and help you with the resolution? — Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fazure-load-testing%2Fissues%2F116%23issuecomment-1305151870&data=05%7C01%7C%7Cf95dffeb90ca4d2a2bdf08dac08b1038%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638034000747948184%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=7qPL7W9pOaIdn7gKvo4SMuwQAc5WGNqKj9kR5YCAUwE%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAKUTMGHYXQGPSFH4TEIRUUTWHCQARANCNFSM6AAAAAARVMP4SI&data=05%7C01%7C%7Cf95dffeb90ca4d2a2bdf08dac08b1038%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638034000747948184%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=b1xngEWpDQYiCWBPCvjN9nuESviJFyBj%2F7nqrlXT5cw%3D&reserved=0>. You are receiving this because you were mentioned.Message ID: ***@***.***>

Sure. Will check and get back to you ASAP

**Comment by Nagarjuna-Vipparthi** [comment]
> Thanks for helping resolve this issue right away. The case number 2211040040006460 It has BEEN MANY DAYS, and no one is doing anything about. We can't increase the count of instance bigger than 1. Also, the current limit of 45 instances is ridiculous, this needs to be addresses ASAP.
> […](#)
> ________________________________ From: Nagarjuna Vipparthi ***@***.***> Sent: Monday, November 7, 2022 1:41 AM To: microsoft/azure-load-testing ***@***.***> Cc: VFXpro ***@***.***>; Mention ***@***.***> Subject: Re: [microsoft/azure-load-testing] [Feature Request] - Please rapidly allow more instance so we can get our 50,000 user instead of 11,250. (Issue #116) @VFXpro<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FVFXpro&data=05%7C01%7C%7Cf95dffeb90ca4d2a2bdf08dac08b1038%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638034000747948184%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lOLy9bussIVeWHVeQDyg5OHoT3KVWyBOPw0NDWQeK%2F0%3D&reserved=0> Can you please provide the ticket numbers so that we can follow up and help you with the resolution? — Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fazure-load-testing%2Fissues%2F116%23issuecomment-1305151870&data=05%7C01%7C%7Cf95dffeb90ca4d2a2bdf08dac08b1038%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638034000747948184%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=7qPL7W9pOaIdn7gKvo4SMuwQAc5WGNqKj9kR5YCAUwE%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAKUTMGHYXQGPSFH4TEIRUUTWHCQARANCNFSM6AAAAAARVMP4SI&data=05%7C01%7C%7Cf95dffeb90ca4d2a2bdf08dac08b1038%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638034000747948184%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=b1xngEWpDQYiCWBPCvjN9nuESviJFyBj%2F7nqrlXT5cw%3D&reserved=0>. You are receiving this because you were mentioned.Message ID: ***@***.***>

Sure. Will check and get back to you ASAP

**Comment by Nagarjuna-Vipparthi** [comment]
The quota limit for engine instances has been increased to 45. Please let us know if you are still facing issues.
